### DISCUSSION TITLE
Github / OpenAI centric tool forever?

### DISCUSSION THREAD

**Discussion by smedegaard** [zed-industries/zed#7927] [Feature Requests]
Open AI is cool , GH copilot are cool. But there are other tools out there. Is it the plan to keep Zed as tightly coupled to Github and OpenAI as it currently is?

**Comment by SomeoneToIgnore** [comment]
Zed does not commit to using any specific tool from that field.
In fact, https://github.com/zed-industries/zed/pull/7484 had landed and, as https://github.com/zed-industries/zed/issues/7565 states, should allow using different tools that are compatible with such format.

The rest of the tools are welcome for integration too, but so far we have not received any PRs on that topic and do not focus on adding those ourselves — but something might change in the future, maybe even this year.
